### PR TITLE
fix: use correct revision for src container tag

### PR DESCRIPTION
**Pull request by scoheb**

### CODE REVIEW

**Review by happybhati — DISMISSED** [review]

**Comment by happybhati** [comment]
Small comment (@scoheb)  README.md and the task version also need to be updated.

**Review by happybhati — DISMISSED** [review]

File: tasks/push-snapshot/README.md [hunk 1]
```diff
@@ -11,6 +11,9 @@ Tekton task to push snapshot images to an image registry using `cosign copy`.
 | pushSourceContainer| Push the source container to the mapped target                            | Yes      | false                |
 | retries            | Retry copy N times                                                        | Yes      | 0                    |
 
+## Changes since 1.1.0
+* The source container source pullspec was updated to use a git sha tag instead of the image digest.
```
Comment (mmalina) [inline tasks/push-snapshot/README.md:15]: What was the issue with using image digest? Was it in fact the digest of the binary container, so it wouldn't work for the source container?

**Review by mmalina — DISMISSED** [review]

**Review by johnbieren — COMMENTED** [review]
The determination of `sha` should probably go back inside the if statement on L137 now

**Comment by happybhati** [comment]
> The determination of `sha` should probably go back inside the if statement on L137 now

@scoheb For more reference https://github.com/redhat-appstudio/release-service-catalog/commit/995c3d8abde65f4c39e5154a8e41b104e5849c50#diff-d3b8ab085f6a1d2f1ebdd5ebc73681bc750fe7b5eb55bcbe45851d2c086b621dL127

File: tasks/push-snapshot/README.md (hunk 1 above)
Comment (scoheb) [reply tasks/push-snapshot/README.md:15]: prior to this PR, we were trying to copy an image using the pullspec `image@sha256:fbe973077b00242eeb3d25101def1be286c621436c5a003527736eabb48f0398-src` which does not makes sense or exist.

what does exist is `image:<git-sha>-src`

**Review by johnbieren — APPROVED** [review]

**Pull request merged**